source/_patterns/02-organisms/components/hero-banner.yaml

```diff
@@ -0,0 +1,53 @@
+$schema: http://json-schema.org/draft-04/schema#
+type: object
+properties:
+  headingId:
+    type: string
+    minLength: 1
+  image:
+    allOf:
+      - $ref: ../../00-atoms/components/picture.yaml
+  subjects:
+    type: object
+    properties:
+      list:
+        type: array
+        minItems: 1
+        items:
+          type: object
+          properties:
+            name:
+              type: string
+              minLength: 1
+            url:
+              type: string
+              minLength: 1
+          required:
+            - name
+            - url
+    required:
+      - list
+  title:
+    type: string
+    minLength: 1
+  titleLength:
+    enum:
+    - small
+    - medium
+    - large
+  url:
+    type: string
+    minLength: 1
+  summary:
+    type: string
+    minLength: 1
+  authors:
+    type: string
+    minLength: 1
+  meta:
+    $ref: ../../01-molecules/components/meta.yaml
+required:
+  - title
+  - url
+  - summary
+  - authors
```

Comment:
Authors should be optional because the content occupying the hero banner may be other content than article or reviewed preprint and therefore may not have authors.

With the authors being optional we could introduce them in journal only when appropriate given the content type.

Reply:
Authors are no longer required